apis/extensions/v1alpha1/addon_types.go

```diff
@@ -80,6 +80,10 @@ type AddonSpec struct {
 	//
 	// +optional
 	CliPlugins []CliPlugin `json:"cliPlugins,omitempty"`
+
+	// Specify all addons that this addon depends on.
+	// +optional
+	AddonDependencies []AddonDependency `json:"addonDependencies,omitempty"`
```

Comment:
What does it mean for a dependency to have an addon's name and version? Based on the issue corresponding to the PR, apecloud-mysql actually depends on a specific version of the etcd cmpd name, but the etcd addon name is not equal to the etcd cmpd name.